src/test/java/lithium/university/TwitterTest.java

```diff
@@ -0,0 +1,174 @@
+package lithium.university;
```

Comment:
i can't tell what is being tested from a class perspective.  Please separate out the tests based on classes and i'll look more in-depth after that.